src/gui/src/mainWindow.cpp

```diff
@@ -593,6 +598,10 @@ void MainWindow::createActions()
   show_dbu_->setCheckable(true);
   show_dbu_->setChecked(false);
 
+  show_db_view_ = new QAction("Show DB view", this);
```

Comment:
Show polygon decomposition.  Preferably not a menu choice as it only useful to developer.